core/math/geometry_3d.cpp

```diff
@@ -775,6 +775,8 @@ Vector<Plane> Geometry3D::build_box_planes(const Vector3 &p_extents) {
 }
 
 Vector<Plane> Geometry3D::build_cylinder_planes(real_t p_radius, real_t p_height, int p_sides, Vector3::Axis p_axis) {
+	ERR_FAIL_COND_V(!(p_axis == Vector3::AXIS_X || p_axis == Vector3::AXIS_Y || p_axis == Vector3::AXIS_Z), Vector<Plane>());
```

Comment:
That's a bit verbose, I'd simplify to:
```cpp
ERR_FAIL_INDEX_V(p_axis, 3, Vector<Plane>());
```

There's plenty occurrence of 3 dimensions being hardcoded/expected in this file so the magic number is not too bad.